src/component/books/Books.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AddBooks from './AddBooks';
+
+const Books = () => (
+  <div>
+    <div className="booklist" />
+    <button type="button">Remove Book</button>
+    <AddBooks />
+  </div>
+);
```

Comment:
- [ ] The Books/BookList component should render a list of books using the `Book` component. Kindly, add the `Book` component then update your book list.